tests/unittests/Makefile

```diff
@@ -4,7 +4,14 @@
 
 include $(CLEAR_VARS)
 
+TESTS_TO_BUILD := Foundation
+
+# Makefile Entrypoint: for running `make` without an argument.
+.PHONY: all
+all: $(foreach t,$(TESTS_TO_BUILD),$(t).UnitTests.exe)
+
```

Comment:
Gladly! :)

---
In reply to: [72871659](https://github.com/Microsoft/WinObjC/pull/726#discussion_r72871659) [](ancestors = 72871659)